feature_album/src/main/java/com/igorwojda/showcase/feature/album/presentation/albumlist/AlbumListFragment.kt

```diff
@@ -23,24 +24,20 @@ class AlbumListFragment : BaseContainerFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        val context = checkNotNull(context)
-
-        albumAdapter.setOnDebouncedClickListener {
-            val navDirections = AlbumListFragmentDirections.actionAlbumListToAlbumDetail(it.artist, it.name, it.mbId)
-            findNavController().navigate(navDirections)
-        }
-
         recyclerView.apply {
             setHasFixedSize(true)
-            val columnWidth = context.resources.getDimension(R.dimen.image_size).toInt()
             layoutManager =
                 GridAutofitLayoutManager(
-                    context,
-                    columnWidth
+                    requireContext(),
```

Comment:
It's a bit hard to spot it, but there is a bit of inconsistency on Kotlin vs Android.

In Kotlin you have two kinds of methods:
`requite()`/`requireNotNull()` that may throw in `IllegalArgumentException`
`check()`/`checkNotNull()` that may throw in `IllegalStateException`

vs in Android you have
`requireContext()` that may throw in `IllegalStateException` (ideally it should be named `checkContext`)

It's quite common to verify arguments and state across project, so I think that distinction is nice and thus I prefer to follow Kotlin approach across the project (to keep it consistent).

I guess I we could remove this local `context` variable, but I would prefer to use `checkNotNull(context)`